src/ws/session.c

```diff
@@ -691,12 +691,17 @@ main (int argc,
   for (i = 0; env_saved[i] != NULL; i++)
     pam_putenv (pamh, env_saved[i]);
 
+  if (pam_putenv (pamh, "COCKPIT_LOGIN_MESSAGES_MEMFD=4") != PAM_SUCCESS)
+    assert (0);
```

Comment:
Could this give a more useful error message, like errx()?

Reply:
Okay.